tests.py

```diff
@@ -81,5 +104,33 @@ def test_subclauses_arithmetic_booleans(self):
         self.run_vsm('subclauses_arithmetic_booleans', input_data,
                 expected_output, False)
 
+
+class TestIPC(TestVSM):
```

Comment:
This is just a different way of duplicating the tests, though, so it doesn't address the concern =).